tips/totw/55.md

```diff
@@ -0,0 +1,104 @@
+---
+title: Tip of the Week #55: Name Counting and unique_ptr
+layout: tips
+sidenav: side-nav-tips.html
+type: markdown
+---
+
+posted 2013-09-12
+
+updated 2017-10-12
```

Comment:
Vaguely prefer full text dates "Oct 12, 2017" because we aren't exactly sorting these, but your call.